src/Core/MailTemplates/Handlebars/Layouts/Full.html.hbs

```diff
@@ -131,7 +131,7 @@
                                 </tr>
                                 <tr style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
                                     <td class="content-block" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 14px; color: #666666; line-height: 25px; margin: 0; -webkit-font-smoothing: antialiased; padding: 15px 0 0 0; -webkit-text-size-adjust: none; text-align: center;" valign="top">
-                                        &copy; 2021 Bitwarden Inc.    
+                                        &copy; {{{CurrentYear}}} Bitwarden Inc.    
```

Comment:
Why 3x `{{{`  vs. 2x `{{` for the token?

Reply:
First reason - following precedent! 

> In Handlebars, the values returned by the {{expression}} are HTML-escaped. Say, if the expression contains &, then the returned HTML-escaped output is genarated as &amp;. If you don't want Handlebars to escape a value, use the "triple-stash", {{{:

In reality, it is not necessary here at all, but it doesn't hurt either.

Reply:
Updated.